compiler-plugin/src/main/kotlin/arrow/meta/quotes/element/ValueArgument.kt

```diff
@@ -0,0 +1,15 @@
+package arrow.meta.quotes.element
+
+import arrow.meta.quotes.Scope
+import org.jetbrains.kotlin.psi.KtExpression
+import org.jetbrains.kotlin.psi.KtValueArgument
+import org.jetbrains.kotlin.psi.KtValueArgumentName
+
+/**
+ * A template destructuring [Scope] for a [ValueArgument]
```

Comment:
All these comments should be replaced with the same examples we are using in the functions